build.gradle.kts

```diff
@@ -36,6 +37,9 @@ dependencies {
     api("org.testcontainers:rabbitmq") {
         because("integration with rabbitmq")
     }
+    api(libs.commons.compress) {
+        because("'1.24.0' version has CVE-2024-25710, CVE-2024-26308 vulnerabilities")
+    }
```

Comment:
question: Where does this dependency come from? Can we update the original dependency?